src/main/java/carpet/CarpetSettings.java

```diff
@@ -283,7 +283,7 @@ private static void set_defaults()
   rule("kelpGenerationGrowLimit", "feature", "limits growth limit of newly naturally generated kelp to this amount of blocks")
                                   .choices("25", "0 2 25").setNotStrict(),
   rule("renewableCoral",          "feature", "Alternative cashing strategy for nether portals"),
-
+  rule("useMainYaw",              "fix", "Use default yaw for facing direction"),
```

Comment:
Please use the convension of adding suffix fix to bugfixes. Also the name should be explanatory wrt the issue is fixing, not the technical implementation, e.g. placementRotationFix or something like that (end user would have no idea what main yaw is and what it fixes, also the description could be more explanatory
```suggestion
  rule("placementRotationFix",              "fix", "fixes block placement rotation issues when player rotates quickly while placing blocks"),
```